src/lxc/log.c

```diff
@@ -502,7 +502,11 @@ static int build_dir(const char *name)
 			continue;
 		*p = '\0';
 
+#ifndef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
 		ret = lxc_unpriv(mkdir(n, 0755));
+#else
+		ret = errno = EEXIST
```

Comment:
looks like CIFuzz isn't happy because of a missing semicolon here
```
2021-03-26T10:38:27.9666314Z [1mlog.c:510:3: [0m[0;1;31merror: [0m[1minvalid operands to binary expression ('int' and 'char *')[0m
2021-03-26T10:38:27.9667081Z                 *p = '/';
2021-03-26T10:38:27.9667577Z [0;1;32m                ^~
2021-03-26T10:38:27.9716900Z [0m1 error generated.
```

Reply:
Yep, thanks caught that too just now!